src/screens/ProfileScreen/elements/BottomSheetBio.js

```diff
@@ -35,7 +36,11 @@ const BottomSheetBio = React.forwardRef((props, ref) => {
             styles={styles.button}
             textStyling={styles.textStyling}
             onPress={props.handleSave}>
-            {props.isLoadingUpdateBio ? <ActivityIndicator size="small" color="#0000ff" /> : 'Save'}
+            {props.isLoadingUpdateBio ? (
+              <ActivityIndicator size="small" color={COLORS.white2} />
+            ) : (
+              'Save'
+            )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The ActivityIndicator's color has been changed from `#0000ff` to `COLORS.white2`. This change should provide better consistency with the overall color scheme of the application. However, ensure that `COLORS.white2` provides enough contrast against its background for accessibility purposes.
```diff
- <ActivityIndicator size="small" color="#0000ff" />
+ <ActivityIndicator size="small" color={COLORS.white2} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->